component/core/src/main/java/org/exoplatform/social/core/space/impl/SpaceTemplateServiceImpl.java

```
@@ -59,7 +70,81 @@ public SpaceTemplateServiceImpl(InitParams params) {
+    for (SpaceTemplate spaceTemplate : getSpaceTemplates(userId)) {
+      String perms = spaceTemplate.getPermissions();
+      StringBuilder templatePermission = new StringBuilder("");
+      if (identity.isMemberOf("/platform/administrators")) {
+        if (perms != null) {
+          ResourceBundle resourceBundle = resourceBundleService.getResourceBundle("locale.portlet.social.SpaceTemplatesPortlet", new Locale(lang));
+          Pattern pattern = Pattern.compile(";");
+          List<String> permissions = pattern.splitAsStream(perms).collect(Collectors.toList());
+          for (String perm : permissions) {
+            try {
+              UserACL.Permission permission = new UserACL.Permission();
+              permission.setPermissionExpression(perm);
+              String groupId = permission.getGroupId();
+              String membership = permission.getMembership();
+              if ("*".equals(membership)) {
+                membership = resourceBundle.getString("social.spaces.templates.any");
+              } else {
+                membership = membership.substring(0, 1).toUpperCase() + membership.substring(1);
+              }
+              GroupHandler groupHandler1 = organizationService.getGroupHandler();
+              String groupName = groupId;
+              Group group = groupHandler1.findGroupById(groupId);
+              if (group != null) {
+                groupName = group.getGroupName();
+              }
+              // Uppercase the first char
+              groupName = groupName.substring(0, 1).toUpperCase() + groupName.substring(1);
+              String key = resourceBundle.getString("social.spaces.templates.permissionEntry");
+              templatePermission.append(key.replace("{0}", membership).replace("{1}", groupName));
+              templatePermission.append(", ");
+            } catch (Exception e) {
+              LOG.warn("Cannot get labelled permission for entry {}", perm, e);
```

We should throw this exception and manage it in UI to display an error

---

ok